boms/cloud-lts-bom/pom.xml

```diff
@@ -367,31 +364,26 @@
       </dependency>
       <dependency>
         <groupId>com.google.cloud</groupId>
-        <!-- BOM? -->
         <artifactId>google-cloud-service-usage-bom</artifactId>
```

Comment:
@Neenu1995 Can I use this BOM to cover API surface?

Reply:
yes